domains/seqyu.json

```
@@ -2,9 +2,9 @@
   "description": "A dumb kid who likes da computer",
   "repo": "https://github.com/nvct-website/nvct-website.github.io",
   "owner": {
-    "username": "Noahvocat (also known as NVCT, Noahvocado or Noahvocados or FrenchGuy or AverageFrenchGuy or AvrgFrnchGuy)",
+    "username": "seqyu",
```

```suggestion
    "username": "nvct-website",
```
This needs to be your Github username.